### PR TITLE
First upload of Faq

### PR DESCRIPTION
**Pull request by Boibit**
Email take 2. I relogged github with this email, so it should be the one.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -2,3 +2,137 @@
...
+We will have a SDK so developers who want to build their pm-based app can plug into our tech stack, and build their applications on the Zeitgeist protocol. We are building applications ourselves too. To top it off we'll be marketing and supporting prediction markets.
+ We also will be supporting select external teams with grants through the Zeitgeist Foundation, a non-profit entity separated from ZeitgeistPM LLC, in a mission to foster healthy growth throughout the Zeitgeist network.
+
+
+unanswered
+==========
+
+Q: How will Zeitgeist try to get enough people to use it, so prediction markets are valid enough and don't suffer from illiquidity? - prediction markets
+
+Q: Where can I find Zeitgeist Documentation? - development
```
Comment (saboonikhil) [inline docs/faq.md:136]: Which documentation are we referring here?

FAQs are going to be part of the documentation right.

File: docs/faq.md [hunk 2]
```diff
@@ -2,3 +2,137 @@
...
+We are building an open and evolving protocol to support prediction markets primitives, including the governance for how this protocol changes over time. This means we are also building an ecosystem for prediction markets and applications based on prediction markets.
+
+We will have a SDK so developers who want to build their pm-based app can plug into our tech stack, and build their applications on the Zeitgeist protocol. We are building applications ourselves too. To top it off we'll be marketing and supporting prediction markets.
+ We also will be supporting select external teams with grants through the Zeitgeist Foundation, a non-profit entity separated from ZeitgeistPM LLC, in a mission to foster healthy growth throughout the Zeitgeist network.
+
+
+unanswered
+==========
+
+Q: How will Zeitgeist try to get enough people to use it, so prediction markets are valid enough and don't suffer from illiquidity? - prediction markets
```
Comment (saboonikhil) [inline docs/faq.md:134]: Not sure if we have an answer yet. Anyone wants to try @maltekliemann @sea212 ?

File: docs/faq.md [hunk 3]
```diff
@@ -2,3 +2,137 @@
...
+
+Q: Where can I find Zeitgeist Documentation? - development
+
+Q: What is the Zeitgeist commitment to transparency? - tokenomics
```
Comment (saboonikhil) [inline docs/faq.md:138]: Should we use https://zeitgeist.pm/ztg as a reference here?

**Comment by saboonikhil** [comment]
@Whisker17 Can we use these FAQs in our documentation? Any idea why this branch hasn't been deployed?

**Comment by maltekliemann** [comment]
@Whisker17 @saboonikhil I remember talking about this with Bob. I believe this is all outdated, so I'm closing these PRs. But if you find anything that you think belongs in the documentation, then just open a new PR.

**Pull request closed without merging**